### PR TITLE
Events on campus location

### PR DESCRIPTION
**Pull request by j-jasik**
Values are reset when the user changes between location options.

Still need to do some testing.

### CODE REVIEW

File: config/dist/config.py.dist [hunk 1]
```diff
@@ -1,72 +0,0 @@
-import os
```
Comment (c-schwarze) [inline config/dist/config.py.dist]: this file (config/dist/config.py.dist) was deleted? Can you add this file back in?

File: tinker/templates/events/form.html [hunk 2]
```diff
@@ -339,19 +339,29 @@ <h3 class="subtitle">{{ field.label }}</h3>
             };
 
 
-            function offCampusLocation() {
+            function offCampusLocation(clear_values) {
+                if (clear_values){ //Clears other options if the user switches in between them
+                    $("#off_campus_location").val('');
+                    $("#on_campus_location").val('none');
```
Comment (c-schwarze) [inline tinker/templates/events/form.html]: why are you setting this as 'none'? just curious

File: tinker/templates/events/form.html [hunk 3]
```diff
@@ -339,19 +339,29 @@ <h3 class="subtitle">{{ field.label }}</h3>
...
                     $("#off_location_wrap").show();
                     $("#on_campus_wrap").hide();
                     $("#other_wrap").hide();
                 }
-                else {
+                else if(val == "Other On Campus"){
                     $("#off_location_wrap").hide();
-                    $("#on_campus_wrap").show();
+                    $("#on_campus_wrap").hide();
                     $("#other_wrap").show();
```
Comment (c-schwarze) [inline tinker/templates/events/form.html]: This is super picky, but can you order these hide/shows for all of these in the same order they appear in the dropdown? (on campas, other on campus, off campus).

File: tinker/templates/events/form.html [hunk 4]
```diff
@@ -339,19 +339,29 @@ <h3 class="subtitle">{{ field.label }}</h3>
             };
 
 
-            function offCampusLocation() {
+            function offCampusLocation(clear_values) {
```
Comment (c-schwarze) [inline tinker/templates/events/form.html]: Can you set a default value to clear_values? It should just be something like:

`function offCampusLocation(clear_values=false) {`

**Comment by c-schwarze** [comment]
@j-jasik For when you get back...

I'm just going through some old PRs. Where is this at?

File: config/dist/config.py.dist (hunk 1 above)
Comment (j-jasik) [reply config/dist/config.py.dist]: The file was put back in.

File: tinker/templates/events/form.html (hunk 2 above)
Comment (j-jasik) [reply tinker/templates/events/form.html]: 'none' is the value for the "-select-" option on the dropdown menu.

File: tinker/templates/events/form.html (hunk 3 above)
Comment (j-jasik) [reply tinker/templates/events/form.html]: Did this.

File: tinker/templates/events/form.html (hunk 4 above)
Comment (j-jasik) [reply tinker/templates/events/form.html]: We tried this out a couple of different ways and decided it was more trouble than it was worth.

**Comment by c-schwarze** [comment]
Looks good

**Pull request merged**